{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T07:34:00.000+0000",
      "latitude": 51.43788110159321,
      "estimatedRestoration": "2022-03-26T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7136'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7136",
      "reference": "JQ7136",
      "name": "TW13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "TW13 6DH"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.3855684695033329,
          51.43788110159321
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-26T10:00:00.000+0000",
      "UUID": "e0b99c5c-82c0-4913-93ba-a06693fda1a4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3855684695033329,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T08:25:00.000+0000",
      "latitude": 50.68641690352284,
      "estimatedRestoration": "2022-03-26T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7146'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7146",
      "reference": "JQ7146",
      "engineerOnSiteTime": "2022-03-26T08:55:00.000+0000",
      "name": "BH20 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH20 4HY",
        "BH20 4HZ",
        "BH20 4JB",
        "BH20 4JQ",
        "BH20 4NR",
        "BH20 4NW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.104458488248723,
          50.68641690352284
        ],
        "type": "Point"
      },
      "UUID": "4fd13e2e-1b35-4e2d-8486-2f478c498d9b",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.104458488248723,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T11:48:00.000+0000",
      "latitude": 51.45906849159711,
      "estimatedRestoration": "2022-03-26T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7193'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7193",
      "reference": "JQ7193",
      "engineerOnSiteTime": "2022-03-26T12:25:00.000+0000",
      "name": "RG4 & RG1 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG1 4PX",
        "RG1 4QE",
        "RG4 8DL",
        "RG4 8DN"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.9640843026777732,
          51.45906849159711
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-26T12:30:00.000+0000",
      "UUID": "42232437-a01c-4024-9c77-5e2f1379151a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9640843026777732,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T13:00:00.000+0000",
      "latitude": 50.72413447588054,
      "estimatedRestoration": "2022-03-26T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205882'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205882",
      "reference": "205882",
      "name": "BH14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH14 0AF",
        "BH14 0NL"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.9549255812813466,
          50.72413447588054
        ],
        "type": "Point"
      },
      "UUID": "03ccee12-fd1a-4ffd-a080-24fc724d97be",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9549255812813466,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T13:52:00.000+0000",
      "latitude": 51.460064815168415,
      "estimatedRestoration": "2022-03-26T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7227'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7227",
      "reference": "JQ7227",
      "name": "RG5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG5 4LE",
        "RG5 4LJ",
        "RG5 4NB"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.9045276126393735,
          51.460064815168415
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-26T14:30:00.000+0000",
      "UUID": "705be0e8-01c5-4e32-aee1-d75c42e957f7",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9045276126393735,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T13:51:00.000+0000",
      "latitude": 51.60324093968918,
      "estimatedRestoration": "2022-03-26T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7224'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7224",
      "reference": "JQ7224",
      "name": "SN25 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN25 4YA",
        "SN25 4YB",
        "SN25 4YD",
        "SN25 4YF",
        "SN25 4YG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.792452040151956,
          51.60324093968918
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-26T15:00:00.000+0000",
      "UUID": "5c5b9235-1e2c-4248-841b-4800a7d01475",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.792452040151956,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T14:03:00.000+0000",
      "latitude": 50.90350195940397,
      "estimatedRestoration": "2022-03-26T15:03:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7231'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7231",
      "reference": "JQ7231",
      "name": "SO14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO14 5AT",
        "SO14 5AW",
        "SO14 5BU",
        "SO14 5BW",
        "SO14 5BX",
        "SO14 5BY",
        "SO14 5FB",
        "SO14 5FE",
        "SO14 5FN",
        "SO14 5FP",
        "SO14 5FQ",
        "SO14 5FR",
        "SO14 5FZ",
        "SO14 5GA",
        "SO14 5GB",
        "SO14 5GL",
        "SO14 5GN",
        "SO14 5GZ",
        "SO14 5JF",
        "SO14 5JP",
        "SO14 5RF",
        "SO15 2EE",
        "SO15 3EB"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.39429019824636,
          50.90350195940397
        ],
        "type": "Point"
      },
      "UUID": "cfe0ce04-ff7d-4d3b-bb39-76c47d591da0",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "459f54c5-5b21-4976-bea1-84777731c09a",
        "ab82cf36-cfdd-4eca-98be-0d00c2e30429",
        "98244aa8-e6ed-4e71-940e-a35b5ab3d803",
        "d7a0fd5e-16c3-4b07-8a87-1d7db0066e3c",
        "550ea32b-20ed-4699-bdf4-875a6782402e",
        "8194b46c-5f10-4c8a-afbf-0aefddb2157d",
        "9a6c5ced-1a57-4ad1-9785-bb82534a6e17",
        "c0d4dd22-15b2-480d-ae50-5e3f17dc31a8",
        "a13d3d68-efc2-4c80-a3b9-c969c8b26c78",
        "9fb76a65-8de0-436a-a2e2-d29de70002b8",
        "90a4135c-4493-4b2d-8a60-326d19e4c2ef",
        "939bae64-41e3-43c9-8860-0715397f773b",
        "0ff2da09-c0c3-4060-b9a7-9b3cb0657553",
        "759585f3-0660-4118-9392-3d664554f72e",
        "0c53b1c1-dc14-45b3-8d8a-f9ab6d1a37ce",
        "86b1a320-fd33-400f-94ac-845d1f8b852a",
        "029ff182-76a7-4c7b-9e65-8b9aa0bbfad9",
        "b753b043-a2df-4d58-9679-3af1076d322e",
        "b9c0f089-d8f6-4bb0-b8b1-bfc75363c75d",
        "873ae4ca-36b0-44a9-98a0-2825bbcfef02",
        "010220aa-de17-489a-b1a4-61c0dc01b72a"
      ],
      "longitude": -1.39429019824636,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T08:24:00.000+0000",
      "latitude": 57.43066334500189,
      "estimatedRestoration": "2022-03-26T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7145'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7145",
      "reference": "JQ7145",
      "name": "AB54 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB54 4PT",
        "AB54 4QT",
        "AB54 6AG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.77628051970065,
          57.43066334500189
        ],
        "type": "Point"
      },
      "UUID": "89efb795-68d2-4ec3-9f74-1af38473ab96",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.77628051970065,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T09:00:00.000+0000",
      "latitude": 56.106070997081595,
      "estimatedRestoration": "2022-03-26T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206756'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206756",
      "reference": "206756",
      "name": "PA31 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PA30 8ER",
        "PA31 8NH",
        "PA31 8NY",
        "PH1 3AF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -4.965601952308764,
          56.106070997081595
        ],
        "type": "Point"
      },
      "UUID": "21219f99-dd22-4963-8bf2-cf0b677a7add",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.965601952308764,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T09:00:00.000+0000",
      "latitude": 56.79820191802763,
      "estimatedRestoration": "2022-03-26T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206723'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206723",
      "reference": "206723",
      "name": "IV63 & PH1 Areas",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "IV63 7YJ",
        "PH1 3AF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -4.132873409813731,
          56.79820191802763
        ],
        "type": "Point"
      },
      "UUID": "a48b5e02-fedb-4507-aa85-457140f51a2c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.132873409813731,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T08:28:00.000+0000",
      "latitude": 56.48632081581416,
      "estimatedRestoration": "2022-03-26T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7147'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7147",
      "reference": "JQ7147",
      "name": "DD4 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "DD4 0BD",
        "DD4 0BE",
        "DD4 0BR",
        "DD4 0BS",
        "DD4 0NX",
        "DD4 0NZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.918866359440038,
          56.48632081581416
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-26T09:10:00.000+0000",
      "UUID": "53909333-190f-44cd-8460-8e2d1c9ad58c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.918866359440038,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T11:18:00.000+0000",
      "latitude": 60.03409283312563,
      "estimatedRestoration": "2022-03-26T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7185'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7185",
      "reference": "JQ7185",
      "name": "ZE2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "ZE2 9HB",
        "ZE2 9HG",
        "ZE2 9HJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.2287695991512158,
          60.03409283312563
        ],
        "type": "Point"
      },
      "UUID": "edd85ef2-10a0-42b0-ade4-78d1d8051051",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2287695991512158,
      "resolved": false
    }
  ]
}
